### PR TITLE
feat(✨): allow using hex strings

### PR DESCRIPTION
**Pull request by felippepuhle**
Closes https://github.com/wcandillon/react-native-redash/issues/88

### CODE REVIEW

File: src/Colors.ts [hunk 1]
```diff
@@ -19,8 +19,44 @@ interface RGBColor {
   r: number;
   g: number;
   b: number;
+  a?: number;
 }
 
+type HEXColor = string;
```
Comment (wcandillon) [inline src/Colors.ts]: I'd be ok to add a dependency with this package instead: https://www.npmjs.com/package/color?

Comment (felippepuhle) [reply src/Colors.ts]: sure, will take a look at it later
thanks!

File: src/Colors.ts [hunk 2]
```diff
@@ -11,16 +12,19 @@ const {
   interpolate,
   divide,
   sub,
-  color,
+  color: reColor,
   Extrapolate
 } = Animated;
 
 interface RGBColor {
```
Comment (wcandillon) [inline src/Colors.ts]: So we can remove this type now no? Thanks for doing this PR!

Comment (felippepuhle) [reply src/Colors.ts]: I don't think so, it's still being used on RGB -> HSV conversion... I don't know what's the use case for HSV color space, so I didn't want to remove it...

Thanks

Comment (wcandillon) [reply src/Colors.ts]: oh right, just saw it's private. Will review this PR in more depth soon.

File: README.md [hunk 3]
```diff
@@ -368,6 +368,25 @@ interpolateColor(
...
+  inputRange: [0, 100],
+  outputRange: ["#c52c27", "#e1b044"]
+  // will become
+  // outputRange: [{ r: 197, g: 43, b: 39 }, { r: 225, g: 176, b: 68 }]
+});
+
+// with alpha
+interpolateColor(node, {
+  inputRange: [0, 100],
+  outputRange: ["#c52c2700", "#c52c27ff"]
```
Comment (wcandillon) [inline README.md]: Holly molly I didn't know about this format with alpha.

**Comment by wcandillon** [comment]
Thank you!

**Pull request merged**